.circleci/config.yml

```diff
@@ -220,8 +220,8 @@ jobs:
       - run:
           name: Set Pipeline Env Vars
           command: |
-            echo "export AWS_ACCESS_KEY_ID=${DEV_AWS_ACCESS_KEY_ID}" >> ${BASH_ENV}
-            echo "export AWS_SECRET_ACCESS_KEY=${DEV_AWS_SECRET_ACCESS_KEY}" >> ${BASH_ENV}
+            echo "export AWS_ACCESS_KEY_ID=${DEV_AWS_ACCESS_KEY_ID2}" >> ${BASH_ENV}
+            echo "export AWS_SECRET_ACCESS_KEY=${DEV_AWS_SECRET_ACCESS_KEY2}" >> ${BASH_ENV}
```

Comment:
There's also `DEV_APP_AWS_SECRET_ACCESS_KEY`

Reply:
this has been added! nice spot btw